resources/test/dumps/Bacteria Hunt Speeds.js

```diff
@@ -165,7 +168,15 @@ var procedures = (function() {
           procedures["MAKE-FLAGELLA"]();
           SelfManager.self().setXY(Prims.randomCoord(world.topology.minPxcor, world.topology.maxPxcor), Prims.randomCoord(world.topology.minPycor, world.topology.maxPycor));
         }, true);
-      }), [1, 2, 3, 4, 5, 6]);
+      }, "[ this-variation ->
```

Comment:
I just wanted to be sure, so I went ahead and ran a manual test with the escaping to check, and I don't think it is necessary.  The output when escaped comes out to the end-user still escaped (`...[ -&gt; ...`), so it'd be confusing for them.  Per Robert's note above, since there isn't an obvious way to turn this into an unintended JS execution, I think we'll be okay.